src/plugin/file/kml/kml.js

```diff
@@ -217,6 +217,42 @@ plugin.file.kml.readStyle = function(node, objectStack) {
 };
 
 
+/**
+ * Parse text field from BalloonStyle tag.
+ * @param {Node} node Node.
+ * @return {string|*}
+ * @private
+ */
+plugin.file.kml.readBalloonText_ = function(node) {
+  return ol.xml.getAllTextContent(node, false).trim();
+};
+
+
+/**
+ * Parse display mode from the BalloonStyle tag.
+ * @param {Node} node Node.
+ * @return {string|*}
+ * @private
+ */
+plugin.file.kml.readDisplayMode_ = function(node) {
```

Comment:
Identical to the previous function. Again, please use `ol.format.XSD.readString`.